client/lib/invites/reducers/invites-validation.js

```diff
@@ -2,22 +2,39 @@
  * External dependencies
  */
 import { fromJS } from 'immutable';
+import mapValues from 'lodash/object/mapValues';
+import pick from 'lodash/object/pick';
 
 /**
  * Internal dependencies
  */
 import { action as ActionTypes } from 'lib/invites/constants';
+import { decodeEntities } from 'lib/formatting';
 
 const initialState = fromJS( {
 	list: {},
 	errors: {}
 } );
 
+function filterObject( object ) {
```

Comment:
Because changing the default behavior of `decodeEntities`, or even added a recursive decode` would require a higher level of testing and discussion, I'll plan on addressing that in a separate PR.